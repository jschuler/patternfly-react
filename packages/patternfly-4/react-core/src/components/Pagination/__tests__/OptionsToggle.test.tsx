// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { OptionsToggle } from '../OptionsToggle';

describe('OptionsToggle test', () => {
  it('OptionsToggle should match snapshot', () => {
    const view = mount(
      <OptionsToggle
        itemsTitle={"'items'"}
				optionsToggle={"'Select'"}
				itemsPerPageTitle={"'Items per page'"}
				firstIndex={0}
				lastIndex={0}
				itemCount={0}
				widgetId={"''"}
				showToggle={true}
				onToggle={(_isOpen: boolean) => undefined as any}
				isOpen={false}
				isDisabled={false}
				parentRef={null}
				toggleTemplate={''}
				onEnter={null}
      />);
    expect(view).toMatchSnapshot();
  });
});
