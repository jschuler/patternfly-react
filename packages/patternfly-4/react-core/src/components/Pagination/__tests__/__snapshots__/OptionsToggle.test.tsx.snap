// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionsToggle test OptionsToggle should match snapshot 1`] = `
<Component
  firstIndex={0}
  isDisabled={false}
  isOpen={false}
  itemCount={0}
  itemsPerPageTitle="'Items per page'"
  itemsTitle="'items'"
  lastIndex={0}
  onEnter={null}
  onToggle={[Function]}
  optionsToggle="'Select'"
  parentRef={null}
  showToggle={true}
  toggleTemplate=""
  widgetId="''"
>
  <div
    className="pf-c-options-menu__toggle pf-m-plain pf-m-text"
  >
    <span
      className="pf-c-options-menu__toggle-text"
    />
    <Component
      aria-label="'Select'"
      className="pf-c-options-menu__toggle-button"
      id="''-toggle"
      isDisabled={true}
      isOpen={false}
      onEnter={null}
      onToggle={[Function]}
      parentRef={null}
    >
      <Toggle
        aria-label="'Select'"
        className="pf-c-options-menu__toggle-button"
        id="''-toggle"
        isActive={false}
        isDisabled={true}
        isFocused={false}
        isHovered={false}
        isOpen={false}
        isPlain={false}
        isPrimary={false}
        isSplitButton={false}
        onEnter={null}
        onToggle={[Function]}
        parentRef={null}
      >
        <button
          aria-expanded={false}
          aria-label="'Select'"
          className="pf-c-dropdown__toggle pf-c-options-menu__toggle-button"
          disabled={true}
          id="''-toggle"
          onClick={[Function]}
          onKeyDown={[Function]}
          type="button"
        >
          <CaretDownIcon
            className=""
            color="currentColor"
            noVerticalAlign={false}
            size="sm"
            title={null}
          >
            <svg
              aria-hidden={true}
              aria-labelledby={null}
              className=""
              fill="currentColor"
              height="1em"
              role="img"
              style={
                Object {
                  "verticalAlign": "-0.125em",
                }
              }
              viewBox="0 0 320 512"
              width="1em"
            >
              <path
                d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"
                transform=""
              />
            </svg>
          </CaretDownIcon>
        </button>
      </Toggle>
    </Component>
  </div>
</Component>
`;
